test/ctia/auth/threatgrid_test.clj

```diff
@@ -0,0 +1,32 @@
+(ns ctia.auth.threatgrid-test
+  (:require [ctia.auth.threatgrid :as sut]
+            [ctia.store-service-test :refer [store-services]]
+            [clojure.test :refer [deftest is testing]]
+            [puppetlabs.trapperkeeper.app :as app]
+            [puppetlabs.trapperkeeper.testutils.bootstrap :refer [with-app-with-config]])
+  (:import [java.util UUID]))
+
+(defn threatgrid-auth-whoami-url-services
+  "Service map for #'sut/threatgrid-auth-whoami-url-service"
+  []
+  {:ThreatgridAuthWhoAmIURLService sut/threatgrid-auth-whoami-url-service})
```

Comment:
Returning maps for the `*-services` functions for easy stubbing and more control over merging service maps.